### PR TITLE
12.7.1 preview

### PR DESCRIPTION
**Pull request by tkyc**

### CODE REVIEW

**Review by barryw-mssql — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,28 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 
+## [12.7.0] Preview Release
```
Comment (Jeffery-Wasty) [inline CHANGELOG.md:6]:
```suggestion
## [12.7.1] Preview Release
```

**Review by Jeffery-Wasty — CHANGES_REQUESTED** [review]

**Review by Jeffery-Wasty — APPROVED** [review]

**Review by barryw-mssql — APPROVED** [review]

**Comment by sschu** [comment]
@tkyc I am wondering why this hasn't landed on mavencentral (see https://mvnrepository.com/artifact/com.microsoft.sqlserver/mssql-jdbc). Does it take so long to be published?

**Comment by tkyc** [comment]
@sschu 

No, I suspect it's because of a release process change. During this release we had to migrate onto a new version of an internal release tool. I'll let you know if anything changes.

**Comment by tkyc** [comment]
@sschu 

This should be resolved. Artifacts will show up sometime tomorrow.

**Comment by sschu** [comment]
@tkyc Nothing to see, yet.

**Comment by tkyc** [comment]
@sschu 

I see the artifact uploaded on my end. But it doesn't look like maven has indexed the artifacts yet. Let's give it a bit more time.

**Comment by sschu** [comment]
@tkyc Still nothing. I wouldn't expect it to take so long.

**Comment by tkyc** [comment]
@tkyc 

I see it there now.

**Comment by tkyc** [comment]
@sschu 

I see it there now.

**Comment by sschu** [comment]
True, thanks!

**Pull request merged**